docs/source/config_file.rst

```diff
@@ -175,6 +175,8 @@ section of the command line docs.
 
     This option may only be set in the global section (``[mypy]``).
 
+    **Note:** On Windows, use UNC paths to avoid using ``:`` (e.g. ``\\127.0.0.1\X$\MyDir`` where ``X`` is the drive letter).
```

Comment:
Thank you for adding this! I think it may be good to use the RST note marker like this:

```suggestion
.. note::
    On Windows, use UNC paths to avoid using ``:`` (e.g. ``\\127.0.0.1\X$\MyDir`` where ``X`` is the drive letter).
```

Reply:
I frankly hesitated, but because it's already in a specific section (the description of `mypy_path`) it could be mistaken for a global note and not a specific `mypy_path` note don't you think?
But OK for me if you prefer 👍

Reply:
Yeah in that case I think it is good as is.